cfme/networks/subnet.py

```diff
@@ -84,6 +121,12 @@ def provider(self):
         from cfme.networks.provider import NetworkProvider
         return parent_of_type(self, NetworkProvider)
 
+    @property
+    def parent_tenant(self):
```

Comment:
We could cache it with `@cached_property` because later we call it twice in a row and it looks like an expensive operation

Reply:
I pushed new changes, this part of the code was changed as it was not working as I was expecting.